src/pages/Question/QuestionPage.tsx

```diff
@@ -57,6 +57,11 @@ export const QuestionPage = (props: IProps) => {
               onUsefulClick={onUsefulClick}
             />
           </Flex>
+          <ModerationStatus
+            status={question.moderation}
+            contentType="howto"
```

Comment:
```suggestion
            contentType="question"
```
copy pasta, as i call it 🍝

Reply:
whoopsie 🍝